### DISCUSSION TITLE
Coherence of "Following" and "Followers" with other platforms.

### DISCUSSION THREAD

**Discussion by kolibril13** [community/community#46559] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Right now, the two menus
* "Following" and "For You"
* "Followers" and "Following"

are the opposite way from twitter and mastodon, as seen on the screenshot below
I did not investigate other social media platforms, but coherence in the order be nice here?  
<img width="703" alt="image" src="https://user-images.githubusercontent.com/44469195/217096168-64cb3443-d3ce-41bc-aeae-0f3632e7daa9.png">

**Comment by Akash1134** [comment, marked as answer]
Hi @kolibril13  ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
